test/functionalTest/cases/950_GET_v2_entity/GET_v2_entity_not_found.test

```diff
@@ -0,0 +1,277 @@
+# Copyright 2015 Telefonica Investigacion y Desarrollo, S.A.U
```

Comment:
Not sure if understanding this comment... maybe this comment is obsolete at c59b819, as I think all the test are ok and their steps make sense. However, if I'm wrong, please drop another comment about the problem.